checklists/apim_checklist.en.json

```diff
@@ -209,6 +211,7 @@
             "text": "Configure autoscaling to scale out the number of instances when the load increases",
             "guid": "bb5f356b-3daf-47a2-a9ee-867a8100bbd5",
             "severity": "Medium",
+            "graph": "resources | where type == "microsoft.apimanagement/service" | join kind = leftouter (resources | where type == "microsoft.insights/autoscalesettings" | extend targetResourceUri = tostring(properties.targetResourceUri)) on $left.id == $right.targetResourceUri | extend compliant = (sku.name == "Premium" and isnotempty(targetResourceUri) and properties1.enabled == true) | distinct id, compliant",            
```

Comment:
Double quotes in the query string break the JSON syntax, please use single quotes